filtered-resource-service/README.md

```diff
@@ -62,26 +62,43 @@ See filtered-resource-service/doc/filtered-resource-service.drawio for the sourc
 The main route of web requests - and subsequent websocket messages - through the service is shown in the above diagram. 
 The green boxes are client web requests, red boxes are kafka topics, and red storage is redis persistence.
 
-### Token-Offset System
+### Token-Offset Actor System
 
 The top-right box is the token-offset-system and its associated worker actors. 
 This performs the task of persisting and retrieving offsets for tokens. 
 We use an Akka Actor System to process both requests from the client and messages from kafka. 
-In the former case, the offset is either looked-up from redis or told to the actor from kafka. 
-In the latter case, the offset is persisted to redis and then told to all running actors for that given token.
+The first diagram below shows what happens when a client connects to the service before the request has been processed by Palisade.
+The second diagram below shows what happens when a client connects to the service after the request has been processed by Palisade.
 
-### Token-Error System
+#### Early Client Connection
```

Comment:
Doesnt this scenario still do a check of Redis to know that it is early?

Reply:
Updated the description and the diagram to reflect this